src/Sylius/Component/Core/Model/OrderItem.php

```diff
@@ -81,13 +81,19 @@ public function getTaxTotal()
     /**
      * {@inheritdoc}
      */
-    public function getSubtotal()
+    public function getDiscountedUnitPrice()
```

Comment:
We should also add some relevant comments to these methods, as it's done [here](https://github.com/Zales0123/Sylius/blob/348885f8364a28ce5c3f35d5b4b617dc1026f1f5/src/Sylius/Component/Core/Model/OrderItem.php#L62).